danuluma/dannstore
# #161213990 add endpoint to post a sale record

#### What does this PR do?
Add an endpoint to create a sale record

#### Description of Task to be completed?
Add the endpoint POST /sales
Write its test

#### How should this be manually tested?
* Set up the app locally as described in the installation documentation
* Use postman to send a POST request to http://localhost:5000/dann/api/v1/sales 
* The POST request should have just the book id in the body with the format of  ```{
        'book_id': int
    }```
* You may now send a GET request to http://localhost:5000/dann/api/v1/sales
* You should get a response containing the details of the sale record created

#### What are the relevant pivotal tracker stories?
[#161213990 ](https://www.pivotaltracker.com/story/show/161213990)
